src/python/antares-xpansion-ui.py

```diff
@@ -0,0 +1,289 @@
+        if self.sequentialRadioButton.isChecked():
+            return "sequential"
+
+    def _get_step(self):
+        for step in self.step_buttons:
+            if self.step_buttons[step].isChecked():
+                return step
+
+    def _get_nb_core(self):
+        return self.nb_core_edit.value()
+
+    def _get_keep_mps_option(self):
+        if self.keep_mps_checkbox.isChecked():
+            return "--keepMps"
+        else:
+            return ""
+
+    def handle_stdout(self):
+        data = self.p.readAllStandardOutput()
+        stdout = bytes(data).decode("utf8")
+        self.message(stdout)
+
+    def handle_stderr(self):
+        data = self.p.readAllStandardError()
+        stderr = bytes(data).decode("utf8")
+        self.message(stderr)
+
+    def handle_state(self, state):
+        if state == QProcess.NotRunning:
+            self._set_run_label()
+            if self._get_step() not in STEP_WITH_SIMULATION_NAME:
+                self._initSimulationNameCombo(self.studyPathTextEdit.text())
+        else:
+            self._set_stop_label()
+
+    def _method_changed(self):
+        self.nb_core_edit.setEnabled(self.mpibendersRadioButton.isChecked())
+
+    def _use_available_core(self):
+        self.nb_core_edit.setValue(os.cpu_count())
+
+    def _set_stop_label(self):
+        self.runButton.setText("Stop")
+        self.runButton.setIcon(QIcon(":/images/stop-48.png"))
+        self.runningLabel.setVisible(True)
+
+    def _set_run_label(self):
+        self.runButton.setText("Run")
+        self.runButton.setIcon(QIcon(":/images/play-48.png"))
+        self.runningLabel.setVisible(False)
+
+    def simulation_name_changed(self, text):
+        for step in self.step_buttons:
+            if text == NEW_SIMULATION_NAME:
+                self.step_buttons[step].setEnabled(step not in STEP_WITH_SIMULATION_NAME)
+            else:
+                self.step_buttons[step].setEnabled(step in STEP_WITH_SIMULATION_NAME)
+
+    def message(self, s):
+        self.logTextEdit.appendPlainText(s.rstrip('\r\n'))
+
+    def cleanup_process(self):
+        self.p.close();
+
+    def run_or_stop(self):
+        if self.p and self.p.isOpen():
+            qm = QMessageBox()
+            ret = qm.question(self, "Stop simulation", "Do you want to stop current simulation ?")
+            if ret == qm.Yes:
+                self.p.close()
+        else:
+            self.run()
+
+    def run(self):
+        self.logTextEdit.clear()
+        study_path = self.studyPathTextEdit.text()
+        install_dir = self.INSTALL_DIR
+        install_dir_full = str(Path(install_dir).resolve())
+
+        program = "antares-xpansion-launcher.exe"
+        commands = ["--installDir", install_dir_full,
+                    "--dataDir", str(study_path),
+                    "--method", self._get_method(),
+                    "--step", self._get_step(),
+                    "-n", str(self._get_nb_core()),
+                    self._get_keep_mps_option()]
```

Comment:
unchecking the option "Keep intermediate files" lead to the error "launch.py: error: unrecognized arguments: "